content/tracing/span/time_events/message_event.md

```diff
@@ -98,6 +98,14 @@ span.add_time_event(time_event.TimeEvent(datetime.datetime.utcnow(), serverEvent
 {{</highlight>}}
 
 {{<highlight js>}}
+const { MessageEventType } = require('@opencensus/core');
+
+// On the client
+span.addMessageEvent(MessageEventType.RECEIVED, 'message_event_test_id');
```

Comment:
Other APIs in the other languages seem to require compressed and uncompressed byte sizes along with the message event ID. How about Node.js?

Reply:
Those fields were recently added but they did not make it to the `addMessageEvent` method unfortunately (mostly my fault as I added the fields and was thinking about this method but did not make a PR for it). You can add message events directly to the `messageEvents` field but then that doesn't handle dropping them when they get too many. 

I think the best thing to do is just update this example again once the `addMessageEvent` method is updated in the next release.

Reply:
Alright, gotcha! Also is that field supposed to be a string or an integer? I ask because for future consistency with all the libraries, Node.js might need to update this code but it is going to break  in the next release then.

Reply:
The message event ID is actually a hexadecimal string, so this example should be modified to be something like `/* id as a hex string */ '1234ffff` or similar instead of `'message_event_id'`.

The `id` field is a 64-bit int in the protos, but the JS `number` type is a double so it's harder to represent 64-bit ints cleanly (before Big Int gets wide adoption). Anyway, I think that's probably why it's a hex string similar to trace ID / span ID in JS. I think `number` would be a cleaner choice if it was started from scratch since it would be easier to increment, but may as well keep it for backwards compatibility unless @mayurkale22 you think otherwise and would want it to be `number`.

I created a PR to add the compressed/uncompressed sizes and a comment clarifying the `id` type so that should be available for the next release. See https://github.com/census-instrumentation/opencensus-node/pull/344

Reply:
>  I think `number` would be a cleaner choice if it was started from scratch since it would be easier to increment, but may as well keep it for backwards compatibility

I completely agreed to @draffensperger's comment. Also proto says it should be unique within a Span (we use ```uuid.v4()``` to generate unique id). For now, I will change ```message_event_id``` to ```/* id as a hex string */ '1234ffff'```.

Reply:
The [OpenCensus gRPC specs](https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/gRPC.md#message-events) talk about message event IDs starting at 1 for both send and received within a connection and incrementing. I believe the purpose is to enable correlation of messages that may have been delayed/sent out of order between the sender and receiver connection assuming both are instrumented with OpenCensus.

Reply:
Thanks for the explanations @draffensperger and @mayurkale22! In that case, let's please update that comment to guide folks and let them know that the value is a hexadecimal id and that it should start from 1. Thanks also for sending the linked PR to add compressed/uncompressed.

Reply:
Done, thanks @odeke-em